### PR TITLE
[updategraph]: Create factory default configuration if graph dhcp not available

### PR DESCRIPTION
**Pull request by Blueve**
#### Why I did it

Factory default configuration should be set if no external configuration source available.
From config-setup design 
https://github.com/Azure/sonic-buildimage/blob/a04da0456bb4af2c38a8021227230b371be81943/files/image_config/config-setup/config-setup#L231
SONiC will generate factory default configuration if graph service not enabled.
The graph service suppose was designed to generate an empty configuration when graph service not available, this could make some troubles like: https://github.com/Azure/sonic-buildimage/issues/9201

#### How I did it
Use platform PRESET value to generate a default configuration instead of generate empty config. 

#### How to verify it

- [ ] To be verify on Nokia 7215 platform

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [x] 202012
- [x] 202106

#### Description for the changelog

Use platform preset to generate default configuration when dhcp graph service is not available 

#### A picture of a cute animal (not mandatory but encouraged)
image

### CODE REVIEW

**Comment by Blueve** [comment]
The build failure can be found in other recent PRs, it should not related to current change.

File: files/image_config/updategraph/updategraph [hunk 1]
```diff
@@ -48,13 +48,13 @@ if [ "$src" = "dhcp" ]; then
...
     if [ "`cat /tmp/dhcp_graph_url`" = "N/A" ]; then
-        echo "No graph_url option in DHCP response. Skipping graph update and generating an empty configuration."
+        echo "No graph_url option in DHCP response. Skipping graph update and generating a factory default configuration."
         PLATFORM=${PLATFORM:-`sonic-cfggen -H -v DEVICE_METADATA.localhost.platform`}
         if [ -f /etc/sonic/minigraph.xml ]; then
             sonic-cfggen -H -m /etc/sonic/minigraph.xml --preset empty > /tmp/device_meta.json
         else
             PRESET=(`head -n 1 /usr/share/sonic/device/$PLATFORM/default_sku`)
-            sonic-cfggen -H -k ${PRESET[0]} --preset empty > /tmp/device_meta.json
+            sonic-cfggen -H -k ${PRESET[0]} --preset ${PRESET[1]} > /tmp/device_meta.json
```
Comment (Blueve) [inline files/image_config/updategraph/updategraph:57]: Instead of change this line, maybe change line 62 will be better?

**Review by yxieca — APPROVED** [review]

**Comment by Blueve** [comment]
This change is not helpful on create default factory configuration.
The updategraph will stuck here forever:
https://github.com/Azure/sonic-buildimage/blob/2e6616f0ceb78eb53ac25961f1e8dd6e96a7bcf4/files/image_config/updategraph/updategraph#L46

**Comment by Blueve** [comment]
This change cannot work as expect due to we will stuck here forever:
https://github.com/Azure/sonic-buildimage/blob/2e6616f0ceb78eb53ac25961f1e8dd6e96a7bcf4/files/image_config/updategraph/updategraph#L46

**Pull request closed without merging**